### PR TITLE
Removes script not found from index

### PR DESCRIPTION
**Pull request by villander**
image

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -21,12 +21,6 @@
     {{content-for "head-footer"}}
   </head>
   <body class="responsive status">
-    <!--[if lt IE 9]>
-    <script type="text/javascript" src="https://emberjs.com/javascripts/common-old-ie-f9a6a6d5.js"></script>
-    <![endif]-->
-    <!--[if gte IE 9]><!-->
-    <script type="text/javascript" src="https://emberjs.com/javascripts/common-modern-8dbdc765.js"></script>
-    <!--<![endif]-->
```
Comment (locks) [inline app/index.html]: Can you remove the two script tags at the bottom, too?

Comment (villander) [reply app/index.html]: Yes, a second

Comment (locks) [reply app/index.html]: Oops, we can only do that once we use ember-styleguide going.

Comment (villander) [reply app/index.html]: This issue already exists #24

**Pull request merged**